test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -569,23 +567,21 @@ public void KeyToVectorWithBagOnnxConversionTest()
             // the associated file in ML.NET repo. Such a comparison can be retired if ONNXRuntime ported to ML.NET
             // can support Linux and Mac.
             var subDir = Path.Combine("..", "..", "BaselineOutput", "Common", "Onnx", "BinaryClassification", "BreastCancer");
-            var onnxTextName = "OneHotBagPipeline.txt";
-            var onnxFileName = "OneHotBagPipeline.onnx";
+            var onnxTextName = "KeyToVector.txt";
+            var onnxFileName = "KeyToVector.onnx";
             var onnxTextPath = GetOutputPath(subDir, onnxTextName);
             var onnxModelPath = GetOutputPath(subDir, onnxFileName);
             SaveOnnxModel(onnxModel, onnxModelPath, onnxTextPath);
-            // Compare results produced by ML.NET and ONNX's runtime.
-            if (IsOnnxRuntimeSupported())
+
+            // Binary OutputKind is currently not supported. 
+            if (IsOnnxRuntimeSupported() && OneHotEncodingEstimator.OutputKind.Binary != outputKind)
             {
                 // Evaluate the saved ONNX model using the data used to train the ML.NET pipeline.
                 var onnxEstimator = mlContext.Transforms.ApplyOnnxModel(onnxModelPath);
                 var onnxTransformer = onnxEstimator.Fit(data);
                 var onnxResult = onnxTransformer.Transform(data);
-                CompareSelectedColumns<float>("Score", "Score", transformedData, onnxResult);
-                CompareSelectedColumns<float>("Probability", "Probability", transformedData, onnxResult);
-                CompareSelectedColumns<bool>("PredictedLabel", "PredictedLabel", transformedData, onnxResult);
+                CompareResults("F2", "F2", transformedData, onnxResult);
             }
-            CheckEquality(subDir, onnxTextName);
```

Comment:
Removing the pipeline to isolate transformer. OneHotEncoding is already used in a pipeline within RemoveVariablesInPipelineTest.